packages/eslint-plugin/docs/rules/ban-ts-comment.md

```diff
@@ -41,13 +41,19 @@ const defaultOptions: Options = {
 
 A value of `true` for a particular directive means that this rule will report if it finds any usage of said directive.
 
-For example, with the defaults above the following patterns are considered warnings:
+For example, with the defaults above the following patterns are considered warnings for single line or comment block lines:
 
 ```ts
 if (false) {
   // @ts-ignore: Unreachable code error
   console.log('hello');
 }
+if (false) {
+  /*
+  @ts-ignore: Unreachable code error
+  */
```

Comment:
the `@ts-ignore` must be on the line before the error, even in a multiline comment

```suggestion
  @ts-ignore: Unreachable code error */
```